rca/project_styleguide/templates/patterns/atoms/modal-launcher/modal-launcher.html

```diff
@@ -1,4 +1,6 @@
-<button data-micromodal-trigger="{{ modal }}" class="modal-launcher">
+
+<button data-micromodal-trigger="{{ item.modal }}" class="modal-launcher"
+    {% if not item.modal %}onClick="parent.location='{{ item.link }}'" {% endif%}>
```

Comment:
@nicklee I've assigned you as a reviewer here so you can check you are happy with how this is done.
if there is no `item.modal` coming for the models context we instead use a default link which goes to a form with the course id passed.

Reply:
looks good to me, we'll be able to use that same logic for adding a modifier to tell the modal launcher whether it needs to be the larger version (larger visusally - when it doesn't link to a modal the action text is a lot longer).

I might end up renaming the modal-launcher, as it's obvious now that this doesn't always launch a modal, but this can be done later.